public/app/percona/tickets/Tickets.styles.ts

```diff
@@ -1,8 +1,18 @@
 import { GrafanaTheme } from '@grafana/data';
 import { css } from '@emotion/css';
+import { palette } from '@grafana/data/src/themes/palette';
```

Comment:
`palette` is part of `GrafanaTheme` passed as parameter to `getStyles`. No need to import